generator/generator.yml

```diff
@@ -63,6 +63,25 @@ modules:
       - 1.3.6.1.4.1.674.10892.5.4 # systemDetailsGroup
       - 1.3.6.1.4.1.674.10892.5.5 # storageDetailsGroup
 
+# HPE MIBs
```

Comment:
this could use a bit more context, given HPE has such a big portfolio a comment on what will be scraped here could help to understand whats happening here.